--- dsdt.dsl	2020-12-30 22:10:06.945246639 +0100
+++ dsdt_fix.dsl	2020-12-30 22:10:02.033535952 +0100
@@ -18,7 +18,7 @@
  *     Compiler ID      "    "
  *     Compiler Version 0x01000013 (16777235)
  */
-DefinitionBlock ("", "DSDT", 1, "LENOVO", "CB-01   ", 0x00000001)
+DefinitionBlock ("", "DSDT", 1, "LENOVO", "CB-01   ", 0x00000002)
 {
     External (_GPE, DeviceObj)
     External (_SB_.ALIB, MethodObj)    // 2 Arguments
@@ -917,20 +917,13 @@
         Zero, 
         Zero
     })
-    If ((CNSB == Zero))
+    Name (_S3, Package (0x04)  // _S3_: S3 System State
     {
-        If ((DAS3 == One))
-        {
-            Name (_S3, Package (0x04)  // _S3_: S3 System State
-            {
-                0x03, 
-                0x03, 
-                Zero, 
-                Zero
-            })
-        }
-    }
-
+        0x03, 
+        0x03, 
+        Zero, 
+        Zero
+    })
     Name (_S4, Package (0x04)  // _S4_: S4 System State
     {
         0x04, 
@@ -11317,11 +11310,6 @@
 
                                 Local0 |= One
                             }
-                            Case (Zero)
-                            {
-                                Local0 |= Zero
-                                Local0 |= 0x02
-                            }
                             Default
                             {
                                 Local0 |= Zero
